Add Q2 2026 newsletter: Territory of Freedom (#894)
OpenSats/website: data/newsletter/2026-Q2.mdx

  />
</a>

<div>

> What Bitcoin did for me is like being that point of light in that dark cave.

[Rajarshi](/blog/developer-spotlight-rajarshi-maitra) walked away from a stable engineering career to build Bitshala, India's
main bitcoin developer education organization.

</div>

</div>

<div className="my-8 flex flex-col gap-4 sm:flex-row sm:items-start sm:gap-5">

<a href="/blog/developer-spotlight-barry-deen" className="block w-full sm:w-56 flex-shrink-0">
  <Image
    src="/static/images/spotlight/barry-deen/featured.jpg"
    alt="Barry Deen"
    width={1200}
    height={675}
    className="w-full h-auto rounded-lg"
  />
</a>

<div>

> The biggest secret is just long hours, attention to detail, and hard work.
> Proof of work.

[Barry](/blog/developer-spotlight-barry-deen), known online as UTXO the
Webmaster, builds Wisp, a nostr client made for people who will never care
what a protocol is.

</div>

</div>

Read the full series on [our blog](/spotlight).

---

## Sats Sent: 3,579,487,680

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis aute irure dolor
in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.

> 

Author: This section is still a work-in-progress.

money is hard. Feels like a treadmill. This is why a large base of recurring
support is so important for our sustainable future. On the bright side, we hold
all donations in bitcoin multisig, which provides us both sovereignty and
purchasing power benefits.

Life is short, the grind is long, the mission is more important than ever. If
you are reading this, do more, whether that is
[supporting our work at OpenSats](/donate) or doing your own thing in parallel
to accelerate freedom tech. We need it. Our
children need it. The future is bright if we make it bright.

If not us, then who? If not now, then when?

Push forward.  
Never stop.

<p className="editors-note__signoff">— <a href="/about/odell">ODELL</a></p>

</div>

<DonateRecurringButtonV2 />

<StatsSentence className="mb-6 text-lg leading-relaxed text-gray-600 dark:text-gray-400" />

---

## Building the Base

Developing software is hard. Even with all the agentic tools that today's
LLM-driven world has to offer, creating and maintaining reliable software is a
difficult and strenuous task.

Any long-lasting endeavor requires a solid foundation, and Bitcoin is no

> exception. OpenSats' mission is to help provide funding so that said foundation

Reviewer: ```suggestion
exception. OpenSats' mission is to help provide funding so that foundation
```

can be built. From [open hardware](/blog/open-hardware-for-open-money) to
[nostr libraries](/blog/the-libraries-behind-open-communication) for open and
censorship-resistant communication, to
[base-layer privacy](/blog/call-for-applications-spring-2026) for bitcoin
itself: OpenSats funds it all.

Whether bull or bear market, we are here to support the builders. Those who are
working on making bitcoin and the internet more open, more resilient, and more
sovereign for those who dare to empower themselves. People like
[Calvin Kim](/blog/calvin-kim-receives-lts-grant), who is working on client
diversity and [Utreexo](/projects/utreexod), and recently joined our
[long-term support roster](/tags/lts) to continue this work.

> The base of any project—any human endeavor—is people. Whether it's software or

Reviewer: ```suggestion
The base of any project, any human endeavor, is people. Whether it's software or
```

Author: https://github.com/OpenSats/website/pull/894/changes#r3671380449

[Rajarshi](/blog/developer-spotlight-rajarshi-maitra) walked away from a stable engineering career to build Bitshala, India's
main bitcoin developer education organization.

</div>

</div>

<div className="my-8 flex flex-col gap-4 sm:flex-row sm:items-start sm:gap-5">

<a href="/blog/developer-spotlight-barry-deen" className="block w-full sm:w-56 flex-shrink-0">
  <Image
    src="/static/images/spotlight/barry-deen/featured.jpg"
    alt="Barry Deen"
    width={1200}
    height={675}
    className="w-full h-auto rounded-lg"
  />
</a>

<div>

> The biggest secret is just long hours, attention to detail, and hard work.
> Proof of work.

[Barry](/blog/developer-spotlight-barry-deen), known online as UTXO the
Webmaster, builds Wisp, a nostr client made for people who will never care
what a protocol is.

</div>

</div>

Read the full series on [our blog](/spotlight).

---

## Sats Sent: 3,579,487,680

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis aute irure dolor
in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.

---

## Software Shipped

Our grantees shipped over 500 releases this quarter, spread across more than

> 90 of the [hundreds of projects](/heartbeat) we support.

Reviewer: > spread across more than 90 of the [hundreds of projects](/heartbeat) we support.

"...we've supported"?

A bit more clarity on this please.

Author: Will reword. I don't like this sentence in the first place.

Some highlights:

- **Splicing you can type:** Core Lightning
  [v26.04](https://github.com/ElementsProject/lightning/releases/tag/v26.04)
  added `splicein` and `spliceout` commands. Last quarter we wrote about
  LTS grantee [Dusty Daemon](/blog/dusty-daemon-receives-lts-grant)

>   getting [splicing](/projects/splicing) into the BOLTs. This quarter,

Reviewer 2: ```suggestion
  getting [splicing](/projects/splicing) into the Lightning specifications (BOLTs). This quarter,
```

Dumbing it down a little.

  node operators can resize a channel with a single command. The same
  release made payments more reliable through parallel pathfinding in

>   askrene.

Reviewer 2: We should find something to explain non-tech people what `askrene` is..
I found this [link](https://blog.blockstream.com/core-lightning-v24-08-steel-backed-up-channels/) which could probably be helpful.

- **[Cashu](/projects/cashu) mints learn on-chain:**
  [CDK v0.17.0](https://github.com/cashubtc/cdk/releases/tag/v0.17.0)
  implements NUT-30, which adds on-chain deposits and withdrawals to
  mints that previously spoke Lightning only.
- **[Floresta](/projects/floresta) and [utreexod](/projects/utreexod):**
  Floresta
  [v0.9.0](https://github.com/vinteumorg/floresta/releases/tag/v0.9.0)
  implements the BIP-183 messaging flow for proof-driven sync, and
  utreexod
  [v0.6.0](https://github.com/utreexo/utreexod/releases/tag/v0.6.0)
  sped up historical proof generation. LTS grantee
  [Calvin Kim](/blog/calvin-kim-receives-lts-grant) leads the utreexod
  work.
- **[Krux](/projects/krux) patches what the robots found:**
  [v26.04.0](https://github.com/selfcustody/krux/releases/tag/v26.04.0)
  fixes a batch of security issues that AI-assisted audits surfaced in

>   the hardware signer, including stricter PSBT checks and protections

Reviewer 2: ```suggestion
  the hardware signer, including stricter [PSBT](/topics/psbt) checks and protections
```

  against malicious QR codes and SD cards.
- **New firmware for the [Bitaxe](/projects/bitaxe):**
  [ESP-Miner v2.14.0](https://github.com/bitaxeorg/ESP-Miner/releases/tag/v2.14.0)
  gives the open-source home miner a customizable dashboard, a
  scoreboard, and logs that survive reboots.

There are too many releases to list, and they only tell one part of the story.
Our grantees also write protocol specifications, do boring maintenance work,
hunt for bugs, and build experimental prototypes of what might come
next.

Some highlights:

> - The LNHANCE Expedition's

Reviewer 2: ```suggestion
- The [LNHANCE](/topics/lnhance) Expedition's
```

  [MCCV](https://github.com/LNHANCE-Expedition/mccv) vault ("More

>   Complicated CTV Vault") can now be tried on signet. This quarter

Reviewer 2: ```suggestion
  Complicated CTV Vault") can now be tried on [signet](/topics/signet). This quarter
```

  brought a working CLI, persistent storage, and a user guide.
- [Smite](https://github.com/morehouse/smite), the Lightning snapshot
  fuzzer we covered [last issue](/newsletter/2026-Q1), gained an IR-based
  fuzzing engine and `smitebot`, a tool that automates long fuzzing
  campaigns.

> - [Bitcoinfuzz](/projects/bitcoinfuzz) added miniscript and descriptor

Reviewer 2: ```suggestion
- [Bitcoinfuzz](/projects/bitcoinfuzz) added [miniscript](/topics/miniscript) and [descriptor](/topics/output-descriptors)
```

>   modules and collected fresh trophies, among them a BIP32 seed length

Reviewer 2: ```suggestion
  modules and collected fresh trophies, among them a [BIP32](/topics/bip-32) seed length
```

  validation bug that affected several libraries.
- [mosaico](https://github.com/pablof7z/mosaico), from LTS grantee
  [Pablo](/blog/pablofz7-receives-lts-grant), is an experimental substrate
  for agent self-organization, similar in spirit to Block's buzz.

Have a look at the [OpenSats heartbeat](/heartbeat) to get a live
feed of every release, commit, and pull request across the projects we
fund and have funded.

<ViewHeartbeatButton />

If you [select our nostr fund](https://heartbeat.opensats.org/?types=release&funds=nostr),
you'll see that this quarter brought new releases for
[Amber](/projects/amber), [Amethyst](/projects/amethyst),
[Applesauce](/projects/applesauce), [Citrine](/projects/citrine),
[Coracle](/projects/coracle), [Jumble](/projects/jumble),
[Mostro](/projects/mostro), [nak](/projects/nak),
[ngit](/projects/ngit), [Wisp](/projects/wisp),
[ZapStore](/projects/zapstore), and more. Wisp reached
[v1.0.0](https://github.com/barrydeen/wisp/releases/tag/v1.0.0) this quarter and
Citrine reached [v3.0.0](https://github.com/greenart7c3/Citrine/releases/tag/v3.0.0),

> both firsts for their projects.

Reviewer 2: This sentence is not totally clear to me. Or better, I get the meaning, but how could a v3 be a first? Maybe something my English is missing..

We have drifted quite far from this vision of the world. In part due to our own negligence, and in part due to the incentives and structures we've built around cyberspace primitives: closed platforms, quarterly profits, walled gardens, top-down access controls, and permissioned networks. The siren song of convenience distracted us from the more lofty ideals of liberty and prosperity. We got user acquisition but lost user freedoms in the process.

It is on us to create a new home for our minds that is worthy, welcoming, and wonderfully joyous to dwell in. We have to build the networks, the railroads, the proverbial information superhighways that allow us to assemble and communicate freely. Structures that are free and open, but also resilient, whether they are inhabited by minds made of flesh or minds made of silicon.

> We have come a long way, but there's much that still needs to be built. As the doors of the legacy internet are closing—with age-verification checks, digital ID, proof-of-human, and similar measures that provide neither freedom nor safety—a new world is emerging.

Reviewer: ```suggestion
We have come a long way, but there's much that still needs to be built. As the doors of the legacy internet are closing, with age-verification checks, digital ID, proof-of-human, and similar measures that provide neither freedom nor safety, a new world is emerging.
```

Reviewer: perhaps a bit nit-picky here

Author: These are my em-dashes. I added them, as a human. 

Fuck me, I'm so annoyed at all this lol

  <Image
    src="/static/images/spotlight/barry-deen/featured.jpg"
    alt="Barry Deen"
    width={1200}
    height={675}
    className="w-full h-auto rounded-lg"
  />
</a>

<div>

> The biggest secret is just long hours, attention to detail, and hard work.
> Proof of work.

[Barry](/blog/developer-spotlight-barry-deen), known online as UTXO the
Webmaster, builds Wisp, a nostr client made for people who will never care
what a protocol is.

</div>

</div>

Read the full series on [our blog](/spotlight).

---

## Sats Sent

In Q2 2026, OpenSats has sent almost 36 BTC to over 300 recipients all over
the world. The transaction fees for these payouts were `19,435` sats, or \$12.51
at current prices. Given that we've sent over \$2.3 million this quarter we
have paid the equivalent of **0.00054% in transaction fees**.

As mentioned in our [last quarterly update](/newsletter/2026-Q1), we have been
sending roughly **\$1,000,000 worth of sats** to our grantees each month. Some
quarters it's a little more, some quarters it's a little less.

Have a look at our [transparency page](/transparency) to dig deeper into our
financial reports or browse our [project pages](/projects) to see how many

> sats they've received in total.

Reviewer 2: ```suggestion
sats each one has received in total.
```

Maybe?

Bitcoin allows us to send money almost instantly, without intermediaries,
without single points of failure, while being incredibly convenient and
cost-efficient. It's a testament to how far we've come.

Sign, broadcast, done.

---

## Software Shipped

Our grantees shipped [over 500 releases](https://heartbeat.opensats.org/?types=release) this quarter, made possible by the funding provided from our [general](/funds/general) and [nostr](/funds/nostr) funds.

Some highlights:

- **Splicing:** Core Lightning
  [v26.04](https://github.com/ElementsProject/lightning/releases/tag/v26.04)
  added `splicein` and `spliceout` commands. Last quarter we wrote about
  LTS grantee [Dusty Daemon](/blog/dusty-daemon-receives-lts-grant)
  getting [splicing](/projects/splicing) into the Lightning specifications (BOLTs). This quarter,
  node operators can resize a channel with a single command. The same
  release made payments more reliable through parallel pathfinding in
  [askrene](https://blog.blockstream.com/core-lightning-v24-08-steel-backed-up-channels/).
- **[Cashu](/projects/cashu) goes on-chain:**
  [CDK v0.17.0](https://github.com/cashubtc/cdk/releases/tag/v0.17.0)

>   implements NUT-30, which adds on-chain deposits and withdrawals to

Reviewer 2: ```suggestion
  implements [NUT-30](https://github.com/cashubtc/nuts/blob/main/30.md),
  which adds on-chain deposits and withdrawals to
```

Link mostly for more tech people that would like to deep dive in this.

  mints that previously spoke Lightning only.
- **[Floresta](/projects/floresta) and [utreexod](/projects/utreexod):**
  Floresta
  [v0.9.0](https://github.com/vinteumorg/floresta/releases/tag/v0.9.0)
  implements the BIP-183 messaging flow for proof-driven sync, and
  utreexod
  [v0.6.0](https://github.com/utreexo/utreexod/releases/tag/v0.6.0)
  sped up historical proof generation. LTS grantee

>   [Calvin Kim](/blog/calvin-kim-receives-lts-grant) leads the utreexod

Reviewer 2: ```suggestion
  [Calvin Kim](/blog/calvin-kim-receives-lts-grant) is leading the utreexod
```

Maybe?

  work.

> - **[Krux](/projects/krux) patched a couple things:**

Reviewer 2: I honestly liked the previous  phrasing

Author: Ok, will revert back to the previous one 👍

---

<div className="editors-note">

Dear Freedom Lovers,

We must build the world we wish to see.

This was the simple premise, the kernel of truth, our north star, that led us
to will OpenSats into existence. An efficient, transparent, sustainable, and
globally scalable nonprofit focused on accelerating freedom technology.
Easier said than done.

The truth is that we were wildly naive on how difficult the task of building
out this mission would be. And yet, here we are, distributing nearly a million
dollars worth of bitcoin to hundreds of grant recipients in over forty
countries every month. Our success is tangible but [the mission](/mission)
remains a challenge, there is plenty of work left to be done.

We could not have accomplished what we have without your support. Thousands of
people have donated. Thousands of builders have applied for grants. The work of
our mostly volunteer team combining the two. We are focused on accelerating
technology, but it is people, motivated individuals, that push this movement
forward.

It should not be a surprise that our pain point is also people. There remains a

> shortage of qualified, responsible, trustworthy volunteers. Working on it. We

Reviewer 3: > Working on it.

"Working on it" instead of something like "`And we're working on that.`"  didn't really flow for me but maybe it's just a me thing

have also had difficulty securing [recurring financial support](/monthly) from both
individuals and companies, many of which benefit from the open source software
we support. Also working on it, but this aspect feels even more uphill. Raising
money is hard. Feels like a treadmill. This is why a large base of recurring
support is so important for our sustainable future. On the bright side, we hold
all donations in bitcoin multisig, which provides us both sovereignty and
purchasing power benefits.

Life is short, the grind is long, the mission is more important than ever. If

> you are reading this, do more, whether that is

Reviewer 3: > If you are reading this, do more, whether that is...

` If you are reading this: do more. Whether that is...`

---
title: 'Sats Well Spent'
issueNumber: 2
date: '2026-07-31'
quarter: '2026-Q2'
headline: 'Territory of Freedom'
summary: 'From nostr foundations to GitHub trending: an inside look at what our grantees have been working on and how permissionless tech creates a new territory of freedom.'
---

<div className="editors-note">

Dear Freedom Lovers,

We must build the world we wish to see.

This was the simple premise, the kernel of truth, our north star, that led us
to will OpenSats into existence. An efficient, transparent, sustainable, and
globally scalable nonprofit focused on accelerating freedom technology.
Easier said than done.

The truth is that we were wildly naive on how difficult the task of building
out this mission would be. And yet, here we are, distributing nearly a million
dollars worth of bitcoin to hundreds of grant recipients in over forty
countries every month. Our success is tangible but [the mission](/mission)
remains a challenge, there is plenty of work left to be done.

We could not have accomplished what we have without your support. Thousands of
people have donated. Thousands of builders have applied for grants. The work of

> our mostly volunteer team combining the two. We are focused on accelerating

Reviewer 3: > The work of our mostly volunteer team combining the two.

`Our team, consisting mostly of volunteer, does the work of combining the two.
`

> technology, but it is people, motivated individuals, that push this movement

Reviewer 3: > but it is people, motivated individuals, that push this movement movement forward.

`but it is the motivated individuals behind the technology that push this movement forward.`

forward.

> It should not be a surprise that our pain point is also people. There remains a

Reviewer 3: > It should not be a surprise that our pain point is also people. There remains a

It should `THEREFORE` not be a surprise that our `MAIN` pain point is also people.

</div>

We receive about a hundred donations each month, and are deeply appreciative of
every single supporter we have. The dream is to receive enough recurring
donations to be sustainable for decades. We are a long stretch away from this
goal, so if you have a minute to [set up a recurring donation](/monthly), please
consider doing so. Every sat counts.

<DonateRecurringButtonV2 />

<StatsSentence className="mb-6 text-lg leading-relaxed text-gray-600 dark:text-gray-400" />

---

## Building the Base

Developing software is hard. Even with all the agentic tools that today's
LLM-driven world has to offer, creating and maintaining reliable software is a
difficult and strenuous task.

Any long-lasting endeavor requires a solid foundation, and Bitcoin is no
exception. OpenSats' mission is to help provide funding so that foundation
can be built. From [open hardware](/blog/open-hardware-for-open-money) to
[nostr libraries](/blog/the-libraries-behind-open-communication) for open and
censorship-resistant communication, to
[base-layer privacy](/blog/call-for-applications-spring-2026) for bitcoin
itself: OpenSats funds it all.

> Whether bull or bear market, we are here to support the builders. Those who are

Reviewer: ```suggestion
Whether it's a bull or bear market, we are here to support the builders. Those who are
```

working on making bitcoin and the internet more open, more resilient, and more
sovereign for those who dare to empower themselves. People like
[Calvin Kim](/blog/calvin-kim-receives-lts-grant), who is working on client
diversity and [Utreexo](/projects/utreexod), and recently joined our
[long-term support roster](/tags/lts) to continue this work.

The base layer of any project—any human endeavor—is people. Whether it's software or
hardware, buildings or protocols. It is people who bring these things into
existence, and it is people who improve and maintain them.

OpenSats' [spotlight](/spotlight) is a series that shines a light on the people we support. What
drives them, what makes them tick, and how come they ended up working on

> freedom tech in the first place?

Reviewer: ```suggestion
drives them, what makes them tick, and how they ended up working on
freedom tech in the first place?
```

  <Image
    src="/static/images/spotlight/rajarshi-maitra/featured.jpg"
    alt="Rajarshi Maitra"
    width={1200}
    height={675}
    className="w-full h-auto rounded-lg"
  />
</a>

<div>

> What Bitcoin did for me is like being that point of light in that dark cave.

[Rajarshi](/blog/developer-spotlight-rajarshi-maitra) walked away from a stable engineering career to build Bitshala, India's
main bitcoin developer education organization.

</div>

</div>

<div className="my-8 flex flex-col gap-4 sm:flex-row sm:items-start sm:gap-5">

<a href="/blog/developer-spotlight-barry-deen" className="block w-full sm:w-56 flex-shrink-0">
  <Image
    src="/static/images/spotlight/barry-deen/featured.jpg"
    alt="Barry Deen"
    width={1200}
    height={675}
    className="w-full h-auto rounded-lg"
  />
</a>

<div>

> The biggest secret is just long hours, attention to detail, and hard work.
> Proof of work.

[Barry](/blog/developer-spotlight-barry-deen), known online as UTXO the
Webmaster, builds Wisp, a nostr client made for people who will never care
what a protocol is.

</div>

</div>

Read the full series on [our blog](/spotlight).

---

## Sats Sent

> In Q2 2026, OpenSats has sent almost 36 BTC to over 300 recipients all over

Reviewer: ```suggestion
In Q2 2026, OpenSats sent almost 36 BTC to over 300 recipients all over
```

  added `splicein` and `spliceout` commands. Last quarter we wrote about
  LTS grantee [Dusty Daemon](/blog/dusty-daemon-receives-lts-grant)
  getting [splicing](/projects/splicing) into the Lightning specifications (BOLTs). This quarter,
  node operators can resize a channel with a single command. The same
  release made payments more reliable through parallel pathfinding in
  [askrene](https://blog.blockstream.com/core-lightning-v24-08-steel-backed-up-channels/).
- **[Cashu](/projects/cashu) goes on-chain:**
  [CDK v0.17.0](https://github.com/cashubtc/cdk/releases/tag/v0.17.0)
  implements [NUT-30](https://github.com/cashubtc/nuts/blob/main/30.md),
  which adds on-chain deposits and withdrawals to
  mints that previously spoke Lightning only.
- **[Floresta](/projects/floresta) and [utreexod](/projects/utreexod):**
  Floresta
  [v0.9.0](https://github.com/vinteumorg/floresta/releases/tag/v0.9.0)
  implements the BIP-183 messaging flow for proof-driven sync, and
  utreexod
  [v0.6.0](https://github.com/utreexo/utreexod/releases/tag/v0.6.0)
  sped up historical proof generation. LTS grantee
  [Calvin Kim](/blog/calvin-kim-receives-lts-grant) is leading the utreexod

>   work.

Reviewer: ```suggestion
  [Calvin Kim](/blog/calvin-kim-receives-lts-grant) is leading the work on utreexod.
```

next.

Some highlights:

- The [LNHANCE](/topics/lnhance) Expedition's
  [MCCV](https://github.com/LNHANCE-Expedition/mccv) vault ("More
  Complicated CTV Vault") can now be tried on [signet](/topics/signet). This quarter
  brought a working CLI, persistent storage, and a user guide.
- [Smite](https://github.com/morehouse/smite), the Lightning snapshot
  fuzzer we covered [last issue](/newsletter/2026-Q1), gained an IR-based
  fuzzing engine and `smitebot`, a tool that automates long fuzzing
  campaigns.
- [Bitcoinfuzz](/projects/bitcoinfuzz) added [miniscript](/topics/miniscript) and [descriptor](/topics/output-descriptors)
  modules and collected fresh trophies, among them a [BIP32](/topics/bip-32) seed length
  validation bug that affected several libraries.
- [mosaico](https://github.com/pablof7z/mosaico), from LTS grantee
  [Pablo](/blog/pablofz7-receives-lts-grant), is an experimental substrate
  for agent self-organization, similar in spirit to Block's buzz.

Have a look at the [OpenSats heartbeat](/heartbeat) to get a live
feed of every release, commit, and pull request across the projects we
fund and have funded.

<ViewHeartbeatButton />

> If you [select our nostr fund](https://heartbeat.opensats.org/?types=release&funds=nostr),

Reviewer 2: This link should be an internal one

Author: Not sure if the forward would pass on the URL params 🤔 

Edit: It does, yay!
